runtime/src/test/scala/com/soundcloud/twinagle/ServerSpec.scala

```diff
@@ -71,6 +73,32 @@ class ServerSpec extends Specification with Mockito {
 
   }
 
+  "generated message filter" in new Context {
+
+    val filter           = mock[MessageFilter]
```

Comment:
do we need to use mocking here? Maybe just create a new instance of `TestFilter` that does what you want?